README.md

```diff
@@ -49,8 +49,11 @@ We defer readers to the arxiv paper to see the effectiveness of these prompts ag
 
 ## Installation
 
-```bash
-$ guardrails hub install hub://arize/dataset_embeddings
+```python
+from guardrails import install
+install("hub://arize-ai/dataset_embeddings_guardrails", quiet=True, install_local_models=True)
```

Comment:
This doesn't look quite right.  The validator is registered as `arize/dataset_embeddings` in the `main.py` file which typically means the hub id should be `hub://arize/dataset_embeddings` unless there are changes elsewhere I'm missing.

Reply:
Hmm in the [Colab](https://colab.research.google.com/github/Arize-ai/dataset-embeddings-guardrails/blob/main/validator/dataset_embeddings_guard.ipynb) this seems to work

<img width="883" alt="Screenshot 2024-08-19 at 7 06 13 PM" src="https://github.com/user-attachments/assets/79e47de3-8538-47a1-ae4b-7dbe8bf8a55f">

Reply:
The other version throws an error

<img width="1230" alt="Screenshot 2024-08-19 at 7 07 11 PM" src="https://github.com/user-attachments/assets/ec4d6a9f-b814-413c-bbd3-8627e674081d">

Reply:
There may be a discrepancy between this and the hub index, let me check

Reply:
Ok, sure enough this is the id listed in the hub index.  It's not the normal pattern but it should work.

Reply:
Ok, let me know if anything needs to be updated on my side!